composer.json

```diff
@@ -56,16 +56,16 @@
     "dompdf/dompdf" : "~2.0.4",
     "firebase/php-jwt": ">=3 <7",
     "rubobaquero/phpquery": "^0.9.15",
-    "symfony/config": "~4.4 || ~6.0",
+    "symfony/config": "~4.4 || ~6.0 || ~7.0",
```

Comment:
For symfony/config, Drupal 10 people get either 6 or 7 depending on their php version. The changelog from 6.4 to 7.0 is almost entirely adding typehints and declarations.

Reply:
For the other symfonys, drupal 10 itself limits to 6 so no change.